nvme-ioctl.c

```diff
@@ -464,6 +464,36 @@ int nvme_get_log(int fd, __u32 nsid, __u8 log_id, bool rae,
 	return 0;
 }
 
+int nvme_get_log_from_uuid(int fd, __u32 nsid, __u8 log_id, bool rae, __u8 uuid_ix,
```

Comment:
This is essentially the same as nvme_get_log(). The only difference is uuid parameter. Could you make reuse of this function instead of duplicating 99% of it?

Reply:
change nvme_get_log to call nvme_get_log_from_uuid